nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -1685,6 +1685,8 @@ The FlowFiles enqueued in a Connection can also be deleted when necessary. The r
 via `Empty queue` in the Connection's context menu. This action can also be performed if the source and destination
 are actively running.
 
+Hovering over the queue will also reveal predicted statistics on when the queue may encounter back pressure, either due to the object count or content size meeting the current threshold
+settings.  Predictions will only be available when NiFi has enough data in it's internal repository and if it's model is accurate enough to broadcast a prediction.
 
```

Comment:
What happens if they aren't available?  Will it tell you?  Can you tell the difference between not enough data and not configured?

Reply:
@ottobackwards thanks for this question, it prompted us to ensure we add enabled/disabled property for the new features (commit has been added to the [PR against master](#3681 3681)).  UI changes are pending (did ask @rfellows offline) but in short tooltip will distinguish when predictions are enabled yet not available vs when predictions are not enabled.